viewmaster/viewmaster.py

```
@@ -235,153 +321,135 @@ def validate_holdings_paths(abspaths):
                 continue
 
             if not os.path.isdir(testpath):
-                LOGGER.error('Not a directory, ignored', testpath)
+                logger.error('Not a directory, ignored', testpath)
                 continue
 
         valid_abspaths.append(abspath)
 
     if not valid_abspaths:
-        raise IOError('Holdings list is empty')
+        raise OSError('Holdings list is empty')
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)